plugins/transport-nio/build.gradle

```diff
@@ -149,10 +149,12 @@ thirdPartyAudit {
         'io.netty.handler.ssl.util.OpenJdkSelfSignedCertGenerator'
     )
 }
-if (project.inFipsJvm == false) {
-    // BouncyCastleFIPS provides this class, so the exclusion is invalid when running CI in
-    // a FIPS JVM with BouncyCastleFIPS Provider
-    thirdPartyAudit.ignoreMissingClasses (
-            'org.bouncycastle.asn1.x500.X500Name'
-    )
+rootProject.globalInfo.ready {
```

Comment:
we already skip third party audit on fips so might as well do this here and for others that would otherwise need `ready`. There's not much to gain from running it if we single out the differences between fips and non fips anyway.